### ISSUE TITLE
How to calculate longitudinal diffusion coefficient of cGMP (DcG)

### ISSUE THREAD

**Issue by larowrow** [rlua/Phototransduction#1]
Hi! I am also interested in 'Spatiotemporal cGMP Dynamics in Living Mouse Rods'. I am wondering if it is possible at all that we can talk about the calculation of the longitudinal diffusion coefficient of cGMP—— DcG? Since eq 12 is a spatial function, but DcG is a constant
![fea65824bd215d294635d182bda82fc](https://user-images.githubusercontent.com/66198349/186165116-816a3521-1c84-4cc4-a303-c80c126051be.png)
.

**Comment by rlua** [comment]
Hi there! I haven't worked on this model for several years and also I am not a coauthor of the paper where this model was presented.
Are you asking how the value of DcG can be estimated? Also I don't see a problem with DcG being a constant in space, if that was what you meant.

**Comment by larowrow** [comment]
Yes, I want to know how the value of DcG can be estimated, but the author donot give me a replay.
